src/Model/Manga/Manga.php

```diff
@@ -189,6 +194,7 @@ public static function fromParser(MangaParser $parser): Manga
         $instance->titleEnglish = $parser->getMangaTitleEnglish();
         $instance->titleSynonyms = $parser->getMangaTitleSynonyms();
         $instance->titleJapanese = $parser->getMangaTitleJapanese();
+        $instance->alternativeTitles = $parser->getAlternativeTitles();
```

Comment:
@nerg4l  Should this be `titleAlternatives` in order to match the naming order of the rest (`titleEnglish`, `titleSynonyms`, `titleJapanese`)?

Which would get auto serialized to `title_alternatives` by the REST API.